cli/cli/commands/service/logs/logs.go

```diff
@@ -34,10 +35,12 @@ const (
 	isEnclaveIdArgGreedy    = false
 
 	serviceIdentifierArgKey        = "service"
-	isServiceIdentifierArgOptional = false
-	isServiceIdentifierArgGreedy   = false
+	isServiceIdentifierArgOptional = true // don't need to pass this in if they use the return all services flag
+	isServiceIdentifierArgGreedy   = true
 
 	shouldFollowLogsFlagKey  = "follow"
+	returnAllServiceLogs     = "all-services"
+	allServicesWildcard      = "*"
```

Comment:
a user requested a wildcard experience like `kurtosis service logs enclave *` where if they passed in the wildcard all service logs would be returned but you hvae to pass it in like `kurtosis service logs enclave '*'` or else my shell tries to interpret the wildcard , lmk if you have better suggestions

Reply:
hmm, not great but nothing better immediately comes to mind

Reply:
maybe a `--all` flag that autocompletes?

Reply:
`-a` , `-all` is already used for "get all logs of the service" right now, so we'll just rock with the wildcard thing for now